python_ta/transforms/type_inference_visitor.py

```diff
@@ -349,10 +349,15 @@ def _assign_tuple(self, target: astroid.Tuple, value: TupleMeta, node: astroid.A
     @accept_failable
     def _lookup_attribute_type(self, node: NodeNG, class_type: type, attribute_name: str) -> TypeResult:
         """Given the node, class and attribute name, return the type of the attribute."""
-        class_name, _, _ = self.get_attribute_class(class_type)
+
```

Comment:
Always check your PRs for unnecessary whitespace before submitting them. Commenting on this issue is not a good use of my time.

Note that there are two occurrences of this in this PR.